### PR TITLE
fix: use JOB_RESOURCE_VARIANT_OVERRIDE to enable resource mode

### PR DESCRIPTION
**Pull request by perangel**
Pass through the relevant resource allocation env vara when running in low resource mode

### CODE REVIEW

**Comment by perangel** [comment]
@gosusnp Can you confirm these are all the vars required?

File: internal/cmd/local/local/cmd.go [hunk 1]
```diff
@@ -356,8 +356,39 @@ func (c *Command) Install(ctx context.Context, opts InstallOpts) error {
...
 		airbyteValues = append(airbyteValues,
-			"global.jobs.resources.requests.cpu=1m",
-			"global.jobs.resources.requests.memory=128m",
+			"server.env_vars.JOB_RESOURCE_VARIANT_OVERRIDE=lowresource",
+			"server.env_vars.JOIN_MAIN_CONTAINER_CPU_LIMIT=0",
+			"server.env_vars.JOIN_MAIN_CONTAINER_CPU_REQUEST=0",
+			"server.env_vars.JOIN_MAIN_CONTAINER_MEMORY_LIMIT=0",
+			"server.env_vars.JOIN_MAIN_CONTAINER_MEMORY_REQUEST=0",
+
+			"worker.env_vars.JOB_RESOURCE_VARIANT_OVERRIDE=lowresource",
```
Comment (gosusnp) [inline internal/cmd/local/local/cmd.go:365]: I don't think we need those in the `worker`

File: internal/cmd/local/local/cmd.go [hunk 2]
```diff
@@ -356,8 +356,39 @@ func (c *Command) Install(ctx context.Context, opts InstallOpts) error {
...
+			"worker.env_vars.JOIN_MAIN_CONTAINER_CPU_LIMIT=0",
+			"worker.env_vars.JOIN_MAIN_CONTAINER_CPU_REQUEST=0",
+			"worker.env_vars.JOIN_MAIN_CONTAINER_MEMORY_LIMIT=0",
+			"worker.env_vars.JOIN_MAIN_CONTAINER_MEMORY_REQUEST=0",
+
+			"workload-launcher.env_vars.JOB_RESOURCE_VARIANT_OVERRIDE=lowresource",
```
Comment (gosusnp) [inline internal/cmd/local/local/cmd.go:371]: `JOB_RESOURCE_VARIANT_OVERRIDE` shouldn't exist in the launcher either.

**Review by gosusnp — APPROVED** [review]
LGTM

**Review by bgroff — APPROVED** [review]

**Pull request merged**